frontend/lib/src/controllers/transaction_controller.dart

```diff
@@ -4,23 +4,37 @@ import 'package:frontend/src/models/transaction.dart';
 import 'package:frontend/src/services/transaction_service.dart';
 
 List<String> transactionTypes = ['Credit', 'Dept', 'Transfer'];
+List<String> transactionCategory = [
```

Comment:
move category and types to /enums/transaction_enum.dart

transfer enums under /models/transaction.dart as well